bindings/bind_py_structure_manager.hh

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file   bind_py_neighbour_manager.hh
+ *
+ * @author Felix Musil <felix.musil@epfl.ch>
+ *
+ * @date   9 Mai 2018
+ *
+ * @brief  File for binding the Neighbour Managers
+ *
+ * Copyright  2018  Felix Musil, COSMO (EPFL), LAMMM (EPFL)
+ *
+ * Rascal is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3, or (at
+ * your option) any later version.
+ *
+ * Rascal is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this software; see the file LICENSE. If not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ */
+
+#ifndef BINDINGS_BIND_PY_STRUCTURE_MANAGER_HH_
+#define BINDINGS_BIND_PY_STRUCTURE_MANAGER_HH_
+
+#include "structure_managers/structure_manager_centers.hh"
+#include "structure_managers/structure_manager_lammps.hh"
+#include "structure_managers/adaptor_strict.hh"
+#include "structure_managers/adaptor_neighbour_list.hh"
+#include "structure_managers/adaptor_increase_maxorder.hh"
+#include "structure_managers/adaptor_half_neighbour_list.hh"
+#include "structure_managers/adaptor_full_neighbour_list.hh"
+#include "structure_managers/make_structure_manager.hh"
+#include "structure_managers/structure_manager.hh"
+#include "structure_managers/structure_manager_base.hh"
+#include "structure_managers/structure_manager_collection.hh"
+
+#include "bind_include.hh"
+
+namespace rascal {
+  void add_structure_managers(py::module &, py::module &);
```

Comment:
...so out of that huge .cc file, only one function is exposed (/declared) here? I don't understand why.

Oh and... yeah, the omitted unused argument thing is bugging me here too.

Reply:
the bindings happen in bind_py_modules.cc where this function is called. What is the problem ?